### DISCUSSION TITLE
Tpot Deployment cross datacenters

### DISCUSSION THREAD

**Discussion by neginsadeghi** [telekom-security/tpotce#1112] [Ideas]
Hello.

Consider that I have Tpot instances on multiple data centers ( Ovh, Hetzener, Worldstream, etc ... )
Now I need a central place for gathering data from all of the instances.

We can say it in this way :
Multiple agents ( sender ) and one server ( receiver )

What we can do to achieve this ?

**Comment by t3chn0m4g3** [comment]
Check out the Readme for [distributed installation](https://github.com/telekom-security/tpotce#t-pot-installer).
